x-pack/plugins/fleet/public/applications/fleet/sections/settings/components/edit_output_flyout/output_form_validators.tsx

```diff
@@ -14,7 +14,10 @@ export function validateHosts(value: string[]) {
   value.forEach((val, idx) => {
     try {
       const urlParsed = new URL(val);
-      if (!['http:', 'https:'].includes(urlParsed.protocol)) {
+      if (!val || val === '') {
```

Comment:
just realized that the null check has to be done before new URL(val) in line 16, otherwise an error will be thrown:
```
new URL('')
VM215:1 Uncaught TypeError: Failed to construct 'URL': Invalid URL
```